src/index.test.ts

```diff
@@ -173,7 +174,7 @@ test('Dependencies range types', async (t) => {
     t.true(range.startsWith(specifier), `Regular dependency ${name} starts with \`${specifier}\`.`)
   }
   for (const [name, range] of Object.entries(ourPeerDeps)) {
-    const specifier = '>='
+    const specifier = '^'
```

Comment:
Why?

Reply:
eslint-config-standard moved peer deps to '^' here: standard/eslint-config-standard@ffc3c75
The tests here require me to use the same versions of peer deps.

Reply:
This change is now reverted by switching to option 2 above